src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -83,9 +94,11 @@ public void stopMotors() {
   public void periodic() {
     motorVelocity = leftShootMotor.getVelocity().getValue();
     motorsAtShootingSpeed = motorVelocity <= targetVelocity + 10 && motorVelocity >= targetVelocity - 10;
+    positionOfArm = pivotMotor.getPosition().getValue() * 360;
     SmartDashboard.putNumber(SMART_DASHBOARD_VELOCITY, motorVelocity);
     SmartDashboard.putNumber(SMART_DASHBOARD_TARGET_VELOCITY, targetVelocity);
-    SmartDashboard.putString(SMART_DASHBOARD_POSITION, positionOfArm.toString());
+    SmartDashboard.putNumber(SMART_DASHBOARD_POSITION, positionOfArm);
+    SmartDashboard.putNumber(SMART_DASHBOARD_TARGET_POSITION, targetArmPosition);
     pivotMotor.setControl(motionMagicVoltage); // -- not controled yet
```

Comment:
We should verify this, but we should also remove this call since if you call `setControl` once it will automatically update at 1khz. We shouldn't need to call `setControl` again unless the position we're aiming for changes.